lib/grpcServer/handlers/core/broadcastTransactionHandlerFactory.js

```diff
@@ -51,16 +51,7 @@ function broadcastTransactionHandlerFactory(insightAPI) {
       throw new InvalidArgumentGrpcError(`invalid transaction: ${transactionIsValid}`);
     }
 
-    let transactionId;
-    try {
-      transactionId = await insightAPI.sendTransaction(serializedTransaction.toString('hex'));
-    } catch (e) {
-      if (e.statusCode === 400) {
-        throw new InvalidArgumentGrpcError(`invalid transaction: ${e.error}`);
-      }
-
-      throw e;
-    }
+    const transactionId = await coreRPCClient.sendRawTransaction(serializedTransaction.toString('hex'));
```

Comment:
But you still need to handle error codes from RPC in case of an invalid transaction and others

Reply:
I implemented error code in DashcoreRPCError in order to deal with that. 
I was then able to detect for -22 and -25